### PR TITLE
NEST-338: Re-implement Azure Reset build with GitHub Actions

### PR DESCRIPTION
**Pull request by DemeSzabolcs**
NEST-338

### CODE REVIEW

**Review by Piedone — CHANGES_REQUESTED** [review]

File: PowerShell/Utilities/Test-Url/Test-Url.psm1 [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<#
```
Comment (DemeSzabolcs) [inline PowerShell/Utilities/Test-Url/Test-Url.psm1:1]: This file is already in `dev` I needed this.

File: .github/actions/initialize/action.yml [hunk 2]
```diff
@@ -17,3 +19,16 @@ runs:
...
+      run: |
+        Install-Module Az.Storage -AllowClobber -Force
+        Install-Module Az.Sql -AllowClobber -Force
+        Install-Module Az.Websites -AllowClobber -Force
+        Install-Module SqlServer -AllowClobber -Force
+
+        foreach ($moduleName in (Get-ChildItem -Directory | select Name))
+        {
+          Import-Module $moduleName.Name
+        }
```
Comment (Piedone) [inline .github/actions/initialize/action.yml:34]:
```suggestion
        {
            Import-Module $moduleName.Name
        }
```

**Review by Piedone — CHANGES_REQUESTED** [review]

**Review by Piedone — APPROVED** [review]

**Comment by Piedone** [comment]
@BenedekFarkas do we still need the feature branch?

**Comment by Piedone** [comment]
Great then!

**Pull request merged**